components/science_archive.md

```diff
@@ -6,8 +6,16 @@ subtitle: Django Application that manages Data products that are stored in AWS S
 
 # Science Archive
 
-This Django application manages the observatory's data products, and references the raw data products which are stored in AWS S3. It features a REST API for retrieving metadata and download links for data products, and also for submitting metadata and data products for upload to S3. For the complete list of filterable FITS header values, check out the [API specification]({% link api/downtime.md %}).
+This Django application manages archival and retrieval of an observatory's data products. It features a REST API for retrieving metadata and download links for data products, and also for submitting metadata and data products for upload to a cloud or local filestore. For the complete list of filterable FITS header values, check out the [API specification]({% link api/science_archive.md %}).
```

Comment:
I guess its not quite correct to say the science archive has a REST api for uploading to a cloud or filestore. Instead maybe we should say it has an API for adding images that have been uploaded to a cloud or local filestore...